src/AspNetCore.Lambda/HttpHandlers.fs

```diff
@@ -140,10 +143,22 @@ let clearResponse =
         ctx.HttpContext.Response.Clear()
         async.Return (Some ctx)
 
+let private getRoute (ctx:HttpContext) =
+    if ctx.Items.ContainsKey RouteKey
+    then
+        let x = ctx.Items.Item RouteKey |> string 
+        if String.IsNullOrEmpty x then None else Some x 
+    else None
+
+let private getRalativePath (ctx:HttpContext) =
```

Comment:
Yeah 😄 good spot. I don't like any of these names, and will be changing them. Suggestions welcome.